spec.html

```diff
@@ -37695,7 +37700,8 @@ <h1>ValidateSharedIntegerTypedArray ( _typedArray_ [ , _waitable_ ] )</h1>
             1. If ! IsUnclampedIntegerElementType(_type_) is *false* and ! IsBigIntElementType(_type_) is *false*, throw a *TypeError* exception.
           1. Assert: _typedArray_ has a [[ViewedArrayBuffer]] internal slot.
           1. Let _buffer_ be _typedArray_.[[ViewedArrayBuffer]].
-          1. If IsSharedArrayBuffer(_buffer_) is *false*, throw a *TypeError* exception.
+          1. Assert: If IsSharedArrayBuffer(_buffer_) is *true*, then IsDetachedBuffer(_buffer_) is *false*.
```

Comment:
would it make more sense to have this assertion inside `IsDetachedBuffer`?

Reply:
Sure.

Reply:
I removed this assert because it turned out to be encoding useless information. IsSharedArrayBuffer already returned *false* of a `null` [[ArrayBufferData]].